### PR TITLE
Updated and added $BAN Information

### PR DESCRIPTION
**Pull request by sebrock**
Line 11: Corrected spelling of 'discord' and updated discord invite link.

### CODE REVIEW

**Review by l33d4n — COMMENTED** [review]
You did not follow the submission guideline:

- uptrennd - Does not support
- TikTok - Does not support
- BitcoinTalk - Does not support

Vite Token Profile Submission Guideline: https://github.com/vitelabs/crypto-info/blob/master/tokens/vite-tutorial.en.md

**Comment by sebrock** [comment]
> You did not follow the submission guideline:
> 
> 
> 
> - uptrennd - Does not support
> 
> - TikTok - Does not support
> 
> - BitcoinTalk - Does not support
> 
> 
> 
> Vite Token Profile Submission Guideline: https://github.com/vitelabs/crypto-info/blob/master/tokens/vite-tutorial.en.md

Thanks for letting me know. I'll correct and resubmit.

**Comment by sebrock** [comment]
Corrected BAN as per contribution guide. @l33d4n would you mind reviewing and merging? thanks!

File: tokens/ban/tti_61f59e574f9f7babfe8908e5.json [hunk 1]
```diff
@@ -6,21 +6,19 @@
...
+        "explorer":["https://creeper.banano.cc/"],
         "twitter":["https://twitter.com/bananocoin"],
         "facebook":["https://www.facebook.com/bananocurrency/"],
+        "reddit":["https://www.reddit.com/r/banano/"],        
         "instagram":["https://www.instagram.com/bananocurrency/"],
-        "whitepaper":["https://banano.cc/yellowpaper/"],
+        "steemit":["https://steemit.com/@bananocurrency"],
         "coinmarketcap":["https://coinmarketcap.com/currencies/banano/"],
-        "publish0x":["https://www.publish0x.com/banano"],
         "youtube":["https://www.youtube.com/c/BananoPictures],
```
Comment (l33d4n) [inline tokens/ban/tti_61f59e574f9f7babfe8908e5.json:18]:
```suggestion
        "youtube":["https://www.youtube.com/c/BananoPictures"],
```

**Review by l33d4n — APPROVED** [review]

File: tokens/ban/tti_61f59e574f9f7babfe8908e5.json [hunk 2]
```diff
@@ -6,15 +6,19 @@
...
+        "explorer":["https://creeper.banano.cc/"],
         "twitter":["https://twitter.com/bananocoin"],
         "facebook":["https://www.facebook.com/bananocurrency/"],
+        "reddit":["https://www.reddit.com/r/banano/"],        
         "instagram":["https://www.instagram.com/bananocurrency/"],
-        "whitepaper":["https://banano.cc/yellowpaper/"],
-        "coinmarketcap":["https://coinmarketcap.com/currencies/banano/"]
+        "steemit":["https://steemit.com/@bananocurrency"],
+        "coinmarketcap":["https://coinmarketcap.com/currencies/banano/"],
+        "youtube":["https://www.youtube.com/c/BananoPictures],
```
Comment (l33d4n) [inline tokens/ban/tti_61f59e574f9f7babfe8908e5.json:18]:
```suggestion
        "youtube":["https://www.youtube.com/c/BananoPictures"],
```

**Review by l33d4n — CHANGES_REQUESTED** [review]

**Comment by l33d4n** [comment]
@sebrock Looks good, just need to add " to the end of the Youtube link

**Comment by sebrock** [comment]
> @sebrock Looks good, just need to add " to the end of the Youtube link

Done. Thanks!

**Review by l33d4n — APPROVED** [review]

**Pull request merged**